locales/en-US/strings.properties

```diff
@@ -193,3 +198,15 @@ error_fallback_default_info=Oops, something went wrong loading this content.
 # that failed to render, and suggests the likely way to deal with the failure:
 # reloading the page.
 error_fallback_default_refresh_suggestion=Refresh page to try again.
+
+# LOCALIZATION NOTE (menu_action_*).  These strings are displayed in the section
+# context menu and are meant as a call to action for the given section.
+menu_action_remove_section=Remove Section
+menu_action_collapse_section=Collapse Section
+menu_action_expand_section=Expand Section
+menu_action_manage_section=Manage {section}
```

Comment:
Same note  about `{section}`, you'd be using a string (name of section) in the middle of a sentence, with potential problems (case, grammatical case and declension). Does it need to be there?

Reply:
Design feedback was to just switch to "Manage Section"

Reply:
I like 👍 😄